lib/DDG/Spice/Currency.pm

```diff
@@ -28,7 +28,8 @@ my $into_qr = qr/\s(?:en|in|to|in ?to|to)\s/i;
 my $vs_qr = qr/\sv(?:ersu|)s\.?\s/i;
 my $question_prefix = qr/(?:convert|what (?:is|are|does)|how (?:much|many) (?:is|are))?\s?/;
 my $number_re = number_style_regex();
-my $cardinal_re = join('|', qw(hundred thousand k million m billion b trillion));
+my $cardinal_re = qr/hundred |thousand |k |million |m |billion |b |trillion /;
```

Comment:
Nope, that will result in `hundred| thousand| k| million| m| billion| b| trillion` not `hundred |thousand |k |million |m |billion |b |trillion`